README.md

```diff
@@ -1 +1,41 @@
-GeForce Now application written in Electron, wrapping around te web browser GFN implementation for ChromeBooks
+# geforcenow-electron
+
+GeForce Now application written in Electron, wrapping around the web browser GFN implementation for ChromeBooks
+
+## How to run
+
+You can run this wrapper directly from a local directory cloned via git. On Arch Linux you can also use the provided PKGBUILD for a system-wide installation.
```

Comment:
The PKGBUILD is in another repository (https://github.com/hmlendea/PKGBUILDs/blob/master/pkg/geforcenow-electron/PKGBUILD)

(Ideally this will be uploaded to the AUR at some point)

Reply:
OK, I'm now referring directly to that one.